test/server/meta.test.js

```diff
@@ -870,3 +870,39 @@ test('should construct a valid script url hosted on www.paypal.cn', () => {
         throw new Error(`Expected script url to be ${ sdkUrl } - got ${ src }`);
     }
 });
+
+test('should error when the script url does not start with "https://"', () => {
```

Comment:
do we already have a test for `http://`?

Reply:
Great point! I updated the test to check for both `http://` and `:https://`. We have code that technically supports checkout.js on `http://` which is why we support both protocols.